### PR TITLE
feat: add Security Controller

### PR DESCRIPTION
**Pull request by nhiv0**
- Access to login page: **/login**
- Access to profile page: **/profile**
- Access to admin page: **/admin**

### CODE REVIEW

File: migrations/Version20220613055604.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE TABLE user (id INT AUTO_INCREMENT NOT NULL, email VARCHAR(180) NOT NULL, roles JSON NOT NULL, password VARCHAR(255) NOT NULL, UNIQUE INDEX UNIQ_8D93D649E7927C74 (email), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
+        $this->addSql('DROP TABLE car');
+    }
+
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE TABLE car (id INT AUTO_INCREMENT NOT NULL, name VARCHAR(255) CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, type VARCHAR(255) CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, brand VARCHAR(255) CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, image VARCHAR(255) CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, price INT NOT NULL, create_at DATETIME DEFAULT NULL COMMENT \'(DC2Type:datetime_immutable)\', PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB COMMENT = \'\' ');
```
Comment (bangdinhnfq) [inline migrations/Version20220613055604.php]: Hmm, pls remove create table when teardown

File: migrations/Version20220613081004.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220613081004 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```
Comment (bangdinhnfq) [inline migrations/Version20220613081004.php:17]: Should write more description

File: templates/admin/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        .example-wrapper code {
+            background: #F5F5F5;
+            padding: 2px 6px;
+        }
+    </style>
+
+    <div class="example-wrapper">
+        <h1>Hello {{ controller_name }}! ✅</h1>
+        <form action="{{ path('logout') }}" method="post" id="form-button-logout">
+            <input type="hidden" name="_csrf_token" value="{{ csrf_token('logout') }}" id="csrf-token"/>
```
Comment (bangdinhnfq) [inline templates/admin/index.html.twig:23]: Good point

File: templates/login/success.html.twig [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html.twig' %}
```
Comment (bangdinhnfq) [inline templates/login/success.html.twig]: Don't need to have login success page

Comment (nhiv0) [reply templates/login/success.html.twig]: i deleted it . thank you so much

**Pull request closed without merging**